### PR TITLE
Added types for Karma 3.0

### PR DESCRIPTION
**Pull request by devoto13**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/karma-runner/karma/blob/master/lib/logger.js#L24 (for the changed bit - `ConfigOptions.loggers`)
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/karma/tslint.json [hunk 1]
```diff
@@ -1,79 +1,6 @@
...
-        "space-before-function-paren": false,
-        "space-within-parens": false,
-        "strict-export-declare-modifiers": false,
-        "trim-file": false,
-        "triple-equals": false,
-        "typedef-whitespace": false,
-        "unified-signatures": false,
-        "void-return": false,
-        "whitespace": false
+        "no-misused-new": false
```
Comment (weswigham) [inline types/karma/tslint.json]: Care to elaborate on what required this exception?

**Review by weswigham — CHANGES_REQUESTED** [review]

File: types/karma/tslint.json (hunk 1 above)
Comment (devoto13) [reply types/karma/tslint.json]: Sure! It fails on https://github.com/DefinitelyTyped/DefinitelyTyped/blob/0028365e1aac60f81bc54e1853f2337650a2fa56/types/karma/index.d.ts#L101

and expects me to convert this to `class`, but converting it to the class will force whole namespace around it to become a value, which in turn will fail declaration merging with https://github.com/DefinitelyTyped/DefinitelyTyped/blob/0028365e1aac60f81bc54e1853f2337650a2fa56/types/karma/index.d.ts#L471

I didn't figure out how to solve this problem without major changes to the structure and decided to leave it for the future.

Comment (weswigham) [reply types/karma/tslint.json]: Hm, ok. Can you use an inline suppression instead of a global disabling of the rule?

Comment (devoto13) [reply types/karma/tslint.json]: Done

**Pull request merged**